src/__internal__/label/label.style.ts

```diff
@@ -1,7 +1,7 @@
 import styled, { css } from "styled-components";
-import PropTypes from "prop-types";
+import { LabelProps } from "./label.component";
 
-const LabelStyle = styled.label`
+const LabelStyle = styled.label<Pick<LabelProps, "isRequired" | "disabled">>`
```

Comment:
what do you think about defining `StyledLabelProps` here and then extending the component interface with it?

Reply:
I prefer the approach you're suggest but I'd like this approach to be adopted in the other components we're refactoring as well just so the codebase is consistent etc. I'll raise this with the rest of the team in slack and make the changes if we get agreement